### PR TITLE
fix: Fix minor bugs in HTML/CSS

### PR DESCRIPTION
**Pull request by MikuroXina**
Fixed some bugs about HTML/CSS [as reported by sonarcloud](https://sonarcloud.io/project/issues?resolved=false&types=BUG&id=bemusic_bemuse).

### Changelog

Fixed some bugs about HTML/CSS as [reported](https://sonarcloud.io/project/issues?issues=AW0WEOc5aLThNPGzLxnI&open=AW0WEOc5aLThNPGzLxnI&id=bemusic_bemuse) [by](https://sonarcloud.io/project/issues?issues=AW0WEOdcaLThNPGzLxnM&open=AW0WEOdcaLThNPGzLxnM&id=bemusic_bemuse) [sonarcloud](https://sonarcloud.io/project/issues?issues=AW0WEOg6aLThNPGzLxn1&open=AW0WEOg6aLThNPGzLxn1&id=bemusic_bemuse).

### CODE REVIEW

File: bemuse/src/app/ui/MusicInfoTabStats.scss [hunk 1]
```diff
@@ -25,7 +25,6 @@
       float: right;
       text-align: right;
       font-size: 1.5em;
-      line-height: 1;
```
Comment (dtinth) [inline bemuse/src/app/ui/MusicInfoTabStats.scss:28]: This affects the layout and is intended. Please revert this change.

File: bemuse/src/app/ui/OptionsCheckbox.scss [hunk 2]
```diff
@@ -22,7 +22,7 @@
       content: '✓';
       color: #b9edff;
       opacity: 0;
-      transform: 0.138s opacity ease-out;
+      transition: 0.138s opacity ease-out;
```
Comment (dtinth) [inline bemuse/src/app/ui/OptionsCheckbox.scss:25]: Nice catch!

**Review by dtinth — APPROVED** [review]

File: bemuse/src/app/ui/MusicInfoTabStats.scss (hunk 1 above)
Comment (MikuroXina) [reply bemuse/src/app/ui/MusicInfoTabStats.scss:28]: line-height: 1em; was specified after two lines. So I thought that there is a duplication. But it is correct to specify a number with no unit, so fixed on 4bd7730b.
